torchinductor/compile_fx.py

```diff
@@ -111,9 +111,22 @@ def cudagraphify(model, inputs, static_input_idxs=()):
     """
     Assumes inputs[static_input_idxs[i]] are always the same memory address
     """
+
+    def static_input(x):
+        # make sure alignment and contiguity of inputs is preserved
+        needed_size = (
+            sum((shape - 1) * stride for shape, stride in zip(x.size(), x.stride())) + 1
+        )
+        needed_size = (needed_size + 31) // 32 * 32
```

Comment:
nit: `utils.ceildiv(needed_size, 32) * 32`

Reply:
lol I didn't know `utils.ceildiv` is in already